x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -309,13 +309,11 @@ export class FleetPlugin
         }),
       esIndexPatternService: new ESIndexPatternSavedObjectService(),
       packageService: {
-        getInstalledEsAssetReferences: async (
+        getInstallation: async (
```

Comment:
any reason to not expose `getInstallation` directly here?

Reply:
great catch, fixed :)